lib/inventory.dart

```diff
@@ -0,0 +1,176 @@
+import 'package:inventory_assistant/misc/api/api_lib.dart' as api;
+import 'package:flutter/material.dart';
+
+class InventoryPage extends StatefulWidget {
+  const InventoryPage({super.key});
+
+  @override
+  State<InventoryPage> createState() => _InventoryPageState();
+}
+
+class _InventoryPageState extends State<InventoryPage> {
+  Map<int, bool> state = {};
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text("Inventory"), centerTitle: true),
+      body: ListView(
```

Comment:
Need some scoll otherwise some stuff can become hidden

Reply:
Added scroll while keeping search at the top